recipes-extended/xen/xen-common.inc

```diff
@@ -94,6 +94,7 @@ SRC_URI_append = " \
     file://v4v.patch \
     file://v4v-viptables.patch \
     file://openxt-xen-xsmv4vuse.patch \
+    file://memory-scrub-on-shutdown.patch \
```

Comment:
suggest naming: `memory-scrub-on-domain-shutdown.patch`
just to make it clearer that this isn't only about host shutdown.